Plugin~/WebRTCPlugin/Codec/EncoderFactory.cpp

```diff
@@ -30,7 +31,11 @@ namespace WebRTC {
         switch (deviceType) {
 #if defined(SUPPORT_D3D11)
             case GRAPHICS_DEVICE_D3D11: {
+#if USE_SOFTWARE_ENCODER

```

Comment:
Please use a variable instead of ifdef.
It will be great if we can set the variable from the Unity side, but it can also be a constant, for now.

Reply:
got it. this variable adds as a static member variable of ContextManager is ok?

Reply:
Ok